pkg/executor/hash_table.go

```diff
@@ -565,7 +611,7 @@ func (es *entryStore) GetStore() (e *entry, memDelta int64) {
 
 type baseHashTable interface {
 	Put(hashKey uint64, rowPtr chunk.RowPtr)
-	Get(hashKey uint64) (rowPtrs []chunk.RowPtr)
+	Get(hashKey uint64) *entry
```

Comment:
Will it be better to add a comment example here for how to iterator the stored ptrs?

Reply:
Added. Nice suggestion.